sycl/test/fpga_tests/global_fpga_device_selector.cpp

```diff
@@ -0,0 +1,26 @@
+//==--- global_fpga_device_selector.cpp - SYCL FPGA device selector test ---==//
+//
```

Comment:
the license section may be omitted for LIT tests.

Reply:
Removed.